modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1064,6 +1064,12 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
       }
 
       const keychain = bitcoin.HDNode.fromBase58(userPrv);
+
+      if (keychain.toBase58() === keychain.neutered().toBase58()) {
+        throw new Error('user private key is only public');
```

Comment:
not: `user private key is only public`/`invalid user private key, a public key was provided` or something along those lines